tutorials/scripting/debug/overview_of_debugging_tools.rst

```diff
@@ -52,26 +53,110 @@ Also, on Android, the game is deployed using the USB cable to speed up deploymen
 Visible Collision Shapes
 ++++++++++++++++++++++++
 
-This option makes collision shapes and raycast nodes visible in the running game.
+When this option is enabled, collision shapes and raycast nodes (for 2D and 3D) will be visible in the running project.
+
+Visible Paths
++++++++++++++
+
+When this option is enabled, curve resources used by path nodes will be visible in the running project.
 
 Visible Navigation
 ++++++++++++++++++
 
-Navigation meshes and polygons will be visible on the running game.
+When this option is enabled, navigation meshes and polygons will be visible in the running project.
 
-Sync Scene Changes
-++++++++++++++++++
+Visible Avoidance
++++++++++++++++++
+
+When this option is enabled, avoidance objects shapes, radius and velocities will be visible in the running project.
```

Comment:
```suggestion
When this option is enabled, avoidance object shapes, radiuses, and velocities will be visible in the running project.
```

Reply:
FYI: This is a direct copy of the tool-tip
https://github.com/godotengine/godot/blob/fe01776f05b1787b28b4a270d53037a3c25f4ca2/editor/plugins/debugger_editor_plugin.cpp#L83

I guess I could fix that too!